addon/mixins/model.js

```diff
@@ -62,19 +67,14 @@ export default Ember.Mixin.create({
 
     if(!deletedRelations.get(relation)) {
       deletedRelations.set(relation, Ember.A());
+      defineProperty(this, `manyToManyDeleted${relation}`, computed.readOnly(`_manyToManyDeleted.${relation}`));
```

Comment:
Can we use `Ember. defineProperty` and `Ember.computed` just so that its consistent with the rest of the code?